tuning/sft_trainer.py

```diff
@@ -306,6 +308,7 @@ def train(
         tokenizer,
         train_args,
         additional_data_handlers,
+        is_multipack=is_multipack,
         is_padding_free=is_padding_free,
```

Comment:
do we have any test which use padding free? will they be affected since you added before padding free? 
could you move this below padding free in that case..